core/query/query.ml

```diff
@@ -521,8 +521,10 @@ struct
     match Tables.lookup Tables.fun_defs f with
     | Some (finfo, (xs, body), z, location) ->
       Some
-      begin
-        match Var.name_of_binder (f, finfo) with
+        begin
+          (* TODO(dhil): This is a bit of an around-about way to
```

Comment:
Nittiest of nit-picks -- round-about, rather than around-about ;P

Is there any way you could package this into a nicer API?

Reply:
Typo, thanks! I think we can do something much nicer. But I would rather keep this particular patch minimal (the whole point of these "small" patches I have been upstreaming over the past year is to implement a big change gradually).